### PR TITLE
Change to instructions for First Web API tutorial

### PR DESCRIPTION
**Pull request by OwainWilliams**
I've updated the install instructions for what packages you need to install.
 
The reason I have added the second lot of instructions is because it wasn't clear you needed to run an install for Microsoft.EntityFrameworkCore.InMemory too. 
The heading only mentioned adding Microsoft.EntityFrameworkCore.SqlServer so once I had did that, I missed the second part of installing Microsoft.EntityFrameworkCore.InMemory.

### CODE REVIEW

File: aspnetcore/tutorials/first-web-api.md [hunk 1]
```diff
@@ -222,13 +222,18 @@ The *database context* is the main class that coordinates Entity Framework funct
...
 * Select the **Browse** tab, and then enter **Microsoft.EntityFrameworkCore.SqlServer** in the search box.
 * Select **Microsoft.EntityFrameworkCore.SqlServer** in the left pane.
 * Select the **Project** check box in the right pane and then select **Install**.
-* Use the preceding instructions to add the `Microsoft.EntityFrameworkCore.InMemory` NuGet package.
+
+* From the **Tools** menu, select **NuGet Package Manager > Manage NuGet Packages for Solution**.
+* Select the **Browse** tab, and then enter **Microsoft.EntityFrameworkCore.InMemory** in the search box.
+* Select **Microsoft.EntityFrameworkCore.InMemory** in the left pane.
+* Select the **Project** check box in the right pane and then select **Install**.
+
```
Comment (scottaddie) [inline aspnetcore/tutorials/first-web-api.md:236]:
```suggestion
```

**Comment by Rick-Anderson** [comment]
@OwainWilliams  @scottaddie  what's left on this PR?

**Comment by scottaddie** [comment]
Thank you for your patience and help with this PR, @OwainWilliams! Your changes will go live within the next 24 hours.

**Comment by OwainWilliams** [comment]
> Thank you for your patience and help with this PR, @OwainWilliams! Your changes will go live within the next 24 hours.

No problem :) I've learned a lot about the styling for these docs so hopefully I can contribute again in the future. 

thanks.

**Pull request merged**